docs/organizations/using-coding-standards.md

```diff
@@ -6,6 +6,8 @@ Coding standards help you ensure that Codacy analyzes multiple repositories with
 
 When you customize the tools or code patterns of a repository that follows a coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
+Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository unassigns any previously applied coding standard.
```

Comment:
I'm not 100% sure about using "unassigns" as it's an unusual word, but I suspect that other variants (overwrites, replaces) may lead the user to believe that something destructive happens to the coding standard itself.

Reply:
Sounds good! :ok_hand: 

The only thing that I'm on the fence about is whether this paragraph could switch positions with the previous one. So we'd talk about high-level information first, and mention the repository-level settings last. But that's just an idea. :bulb:

Reply:
I saw that Vale complains about the word "unassigns", but this should be such a rare word that it's probably better to just ignore it locally:

```suggestion
Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository <span class="skipvale">unassigns</span> any previously applied coding standard.
```